### PR TITLE
feat: sorting by price on collection items page

### LINKED ISSUE #5446: Sorting by price on collection
I thought we have it 👀

https://beta.kodadot.xyz/rmrk2/collection/7e982817a2d2575031-C6N4I?page=1

image

in old koda
image

### PR DESCRIPTION
**Pull request by leo-anderson-x**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #5446
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DzUbHCk3Fr3XdCPNKo7uCJvtBH7YfgiFbn4Gr3VmCMiFy1C)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

https://user-images.githubusercontent.com/128157824/229132971-2ab8b143-55d3-4b73-b771-04b06619631f.mp4

### CODE REVIEW

File: components/explore/ExploreSort.vue [hunk 1]
```diff
@@ -62,7 +62,9 @@ const router = useRouter()
 const { $i18n } = useNuxtApp()
 
 const isActive = ref(false)
-const isItems = computed(() => route.path.includes('items'))
+const isItems = computed(
+  () => route.path.includes('items') || route.path.includes('collection')
```
Comment (leo-anderson-x) [inline components/explore/ExploreSort.vue:66]: There are two places with an items list:
1. /explore/items
2. /collection/xxxx

File: components/explore/ExploreSort.vue [hunk 2]
```diff
@@ -62,7 +62,9 @@ const router = useRouter()
...
+)
 const options = computed(() => {
   return isItems.value
     ? NFT_SQUID_SORT_CONDITION_LIST
```
[the remark this answers is not in the document]
Comment (leo-anderson-x) [reply components/explore/ExploreSort.vue:70]: `ExploreSort` is also used on the `/explore/collectibles` page, and there are actually no price related sorting options on this page, so the conditional judgment here is still needed to separate the sort lists of `collectionListWithSearch` and `nftListWithSearch`.

**Review by vikiival — APPROVED** [review]
Some comment would be on the top of would be nice

if someone creates a page that has either items or collection it would create bug.

Otherwise It does the job

**Comment by vikiival** [comment]
pay 10 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $10 @ 32.55 USD/KSM ~ 0.307 $KSM
🧗 [DzUbHCk3Fr3XdCPNKo7uCJvtBH7YfgiFbn4Gr3VmCMiFy1C](https://kusama.subscan.io/account/DzUbHCk3Fr3XdCPNKo7uCJvtBH7YfgiFbn4Gr3VmCMiFy1C)
🔗 [0xfed7cabde56e48c0757212cb7c42c956444cc7e77f6bb79f9f7eec71a630c99e](https://kusama.subscan.io/extrinsic/0xfed7cabde56e48c0757212cb7c42c956444cc7e77f6bb79f9f7eec71a630c99e)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**